staging/src/kubevirt.io/client-go/api/v1/utils.go

```diff
@@ -46,3 +49,17 @@ func NewMinimalVMIWithNS(namespace, name string) *VirtualMachineInstance {
 	}
 	return vmi
 }
+
+func (c CustomizeComponents) String() string {
+	sort.SliceStable(c.Patches, func(i, j int) bool {
+		return len(c.Patches[i].Patch) < len(c.Patches[j].Patch)
+	})
+
+	s := "CustomizeComponentsPatches"
+
+	for _, p := range c.Patches {
+		s += fmt.Sprintf("Resource%sName%sPatch%sType%s", p.ResourceType, p.ResourceName, p.Patch, string(p.Type))
```

Comment:
I think that in this case using `strings.StringBuilder` will be much more efficient.
Strings are immutable and for every patch we actually reallocate a new memory chunk and reference `s` to the new location.